SongLab/Components/Home/HomeView.swift

```diff
@@ -34,18 +34,21 @@ struct HomeView: View {
                 selectedSession: $viewModel.selectedSession
             )
             VStack {
+                HomeViewNavBarView()
                 Spacer()
                 TrackingToolbarView(
                     audioManager: viewModel.audioManager,
-                    isRecording: $viewModel.isRecording
-                )
-                .ignoresSafeArea()
-                .background(
-                    .ultraThinMaterial
+                    isRecording: $viewModel.isRecording,
+                    isSettingsPresented: $viewModel.isSettingsPresented
                 )
-                .background(Color.black.opacity(0.4))
+                
             }
         }
+        .sheet(isPresented: $viewModel.isSettingsPresented) {
+            AppSettingsView()
+        }
+        .environmentObject(viewModel.appTheme)
```

Comment:
I'd move this to `SongLabApp.swift` since that is the launch point of the app and the theme is global.